.pre-commit-hooks.yaml

```diff
@@ -0,0 +1,6 @@
+- id: commisery
```

Comment:
Added a pre-commit configuration as the new set-up is compatible with pre-commit.